Library/Formula/git-cmd-hub.rb

```diff
@@ -0,0 +1,25 @@
+class GitCmdHub < Formula
+  desc "Interact with GitHub's v3 REST API using the git hub subcommand"
+  homepage "https://github.com/ingydotnet/git-hub"
+  url "https://github.com/ingydotnet/git-hub/archive/0.1.5.tar.gz"
+  sha256 "c057c2c56fa84a389105f935f63a72e5fa3f9fb9412a273d87b403b525ee2225"
+
+  def install
+    inreplace "Makefile" do |f|
+      # Prevent the makefile from installing outside of the prefix
+      f.gsub! "$(shell git --exec-path | sed 's/.*://')", "$(PREFIX)/bin"
+    end
+
+    inreplace "lib/git-hub" do |f|
+      # Fix the hardcoded directory to match Homebrew's structure
+      f.gsub! "SELFDIR=\"$(cd -P `dirname $BASH_SOURCE` && pwd -P)\"", "SELFDIR=#{bin}"
+    end
+
+    system "make", "install", "PREFIX=#{prefix}"
+  end
+
+  test do
+    system "[ -f #{File.join(man1, "git-hub.1")} ]"
+    system "[ -f #{File.join(bin, "git-hub")} ]"
```

Comment:
This is a bit of an odd application regarding tests. All of the real commands require a config file storing github user and authentication information (environment variables may be able to be used instead; looking into this) to be present and start the config wizard if one isn't found. I suppose I could create a user / credentials to use for this purpose, but that seems like a bit of a hack. If you have any suggestions here, I'm all ears.